internal/cmd/service/update/tag/tag.go

```diff
@@ -0,0 +1,125 @@
+package tag
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/zeabur/cli/internal/cmdutil"
+	"github.com/zeabur/cli/internal/util"
+)
+
+type Options struct {
+	id   string
+	name string
+
+	environmentID string
+
+	tag string
+
+	skipConfirm bool
+}
+
+func NewCmdTag(f *cmdutil.Factory) *cobra.Command {
+	opts := &Options{}
+	zctx := f.Config.GetContext()
+
+	cmd := &cobra.Command{
+		Use:   "tag",
+		Short: "Update image tag of a prebuilt service",
+		PreRunE: util.RunEChain(
+			util.NeedProjectContextWhenNonInteractive(f),
+			util.DefaultIDNameByContext(zctx.GetService(), &opts.id, &opts.name),
+			util.DefaultIDByContext(zctx.GetEnvironment(), &opts.environmentID),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runUpdate(f, opts)
+		},
+	}
+
+	util.AddServiceParam(cmd, &opts.id, &opts.name)
+	util.AddEnvOfServiceParam(cmd, &opts.environmentID)
+	cmd.Flags().BoolVarP(&opts.skipConfirm, "yes", "y", false, "Skip confirmation")
+	cmd.Flags().StringVarP(&opts.tag, "tag", "t", "", "The new tag of the image")
```

Comment:
Perhaps it would be better if the default value is `latest`?

Reply:
Good idea https://github.com/zeabur/cli/pull/93/commits/32893af75a03a2b256ac10a35d3bf6b5b44998f2